src/components/Layout/example/Navigation.tsx

```diff
@@ -0,0 +1,41 @@
+import { motion } from 'framer-motion'
+import { MenuItem } from './MenuItem'
+
+import styles from './example.module.scss'
+
+const visibilityVisible = 'visible' as const
+const visibilityHidden = 'hidden' as const
```

Comment:
why do we need this? use inline like `visibility: "visible"`

Reply:
Thank you, done